resources/server/twitter/twitter.service.ts

```diff
@@ -12,8 +14,11 @@ class _TwitterService {
     twitterLogger.debug('Twitter service started');
   }
 
-  async handleGetOrCreateProfile(src: number) {
-    const identifier = PlayerService.getIdentifier(src);
+  async handleGetOrCreateProfile(
+    reqObj: PromiseRequest<void>,
+    resp: PromiseEventResp<Profile | string[]>,
+  ) {
+    const identifier = PlayerService.getIdentifier(reqObj.source);
```

Comment:
This can potentially return null if for some reason our source isn't present in the PlayerService maps. *Might* be worth sanity checking but meh.